hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/metadata/DatanodeStoreSchemaThreeImpl.java

```diff
@@ -88,4 +92,47 @@ public void removeKVContainerData(long containerID) throws IOException {
       getBatchHandler().commitBatchOperation(batch);
     }
   }
+
+  public void dumpKVContainerData(long containerID, File metaDir)
+      throws IOException {
+    String prefix = getContainerKeyPrefix(containerID);
+    getMetadataTable().dumpToFileWithPrefix(
+        getTableDumpFile(getMetadataTable(), metaDir), prefix);
+    getBlockDataTable().dumpToFileWithPrefix(
+        getTableDumpFile(getBlockDataTable(), metaDir), prefix);
+    getDeletedBlocksTable().dumpToFileWithPrefix(
+        getTableDumpFile(getDeletedBlocksTable(), metaDir), prefix);
+    getDeleteTransactionTable().dumpToFileWithPrefix(
+        getTableDumpFile(getDeleteTransactionTable(), metaDir),
+        prefix);
+  }
+
+  public void loadKVContainerData(File metaDir)
+      throws IOException {
+    getMetadataTable().loadFromFile(
+        getTableDumpFile(getMetadataTable(), metaDir));
+    getBlockDataTable().loadFromFile(
+        getTableDumpFile(getBlockDataTable(), metaDir));
+    getDeletedBlocksTable().loadFromFile(
+        getTableDumpFile(getDeletedBlocksTable(), metaDir));
+    getDeleteTransactionTable().loadFromFile(
+        getTableDumpFile(getDeleteTransactionTable(), metaDir));
+  }
+
+  public void cleanupAllDumpFiles(File metaDir)
```

Comment:
Can we more this cleanupAllDumpFiles to BlockUtils.java, and make it a static function? 
And rename it to deleteAllDumpFiles.